<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdTrabajos</class>
 <widget class="QWidget" name="wdTrabajos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>568</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>506</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Trabajos</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Zekton</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Observaciones:</string>
     </property>
     <property name="buddy">
      <cstring>ptObs</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Zekton</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Procesos:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <property name="rowWrapPolicy">
        <enum>QFormLayout::DontWrapRows</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Zekton</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Referencia:</string>
         </property>
         <property name="buddy">
          <cstring>leRef</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="leRef">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Zekton</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Nombre: </string>
         </property>
         <property name="buddy">
          <cstring>leNombre</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="leNombre">
         <property name="maximumSize">
          <size>
           <width>800</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Zekton</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Descripción:</string>
         </property>
         <property name="buddy">
          <cstring>leDescripcion</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="leDescripcion">
         <property name="maximumSize">
          <size>
           <width>800</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton, QLineEdit, QComboBox { color: red }</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Zekton</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Proveedor:</string>
         </property>
         <property name="buddy">
          <cstring>leProveedor</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="HorizontalLayout" stretch="0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLineEdit" name="leProveedor">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbProveedor">
           <property name="maximumSize">
            <size>
             <width>720</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Zekton</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Manufacturado:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
        </layout>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="chbManufacturado">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="taPropiedades">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tbModificadores">
        <attribute name="title">
         <string>Tarificación</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tbTarifas">
        <attribute name="title">
         <string>Page</string>
        </attribute>
        <widget class="QLineEdit" name="leVar1">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>70</width>
           <height>25</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QComboBox" name="cbVar1">
         <property name="geometry">
          <rect>
           <x>101</x>
           <y>40</y>
           <width>720</width>
           <height>25</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>720</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QComboBox" name="cbVar2">
         <property name="geometry">
          <rect>
           <x>101</x>
           <y>70</y>
           <width>720</width>
           <height>25</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>720</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QLineEdit" name="leVar2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>70</width>
           <height>25</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QComboBox" name="cbVar3">
         <property name="geometry">
          <rect>
           <x>101</x>
           <y>100</y>
           <width>720</width>
           <height>25</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>720</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QLineEdit" name="leVar3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>70</width>
           <height>25</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QLineEdit" name="leNumVariantes">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>70</width>
           <height>25</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>81</width>
           <height>27</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Zekton</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Variantes:</string>
         </property>
         <property name="buddy">
          <cstring>leProveedor</cstring>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tbEscandallo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <attribute name="title">
         <string>Escandallo</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetNoConstraint</enum>
           </property>
           <item row="4" column="1">
            <widget class="QTableWidget" name="twTareas">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>Zekton</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Despiece:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QTableWidget" name="twDespiece">
             <property name="maximumSize">
              <size>
               <width>600</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Zekton</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Proceso:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbProcesos"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPlainTextEdit" name="ptObs">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>75</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leNombre</tabstop>
  <tabstop>leDescripcion</tabstop>
  <tabstop>leProveedor</tabstop>
  <tabstop>cbProveedor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
